src/axes/timeAxis.ts

```diff
@@ -491,7 +492,7 @@ export module Axes {
     }
 
     private _hideOverflowingTiers() {
-      let availableHeight = this.height();
+      let availableHeight = this.height() - this.margin();
```

Comment:
The margin is intended to be "soft", meaning that the main visualization (tiers in this case) can steal space from it if they need it.